jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/StatisticsHandler.java

```diff
@@ -36,6 +36,9 @@ public class StatisticsHandler extends EventsHandler
 {
     private final CounterStatistic _requestStats = new CounterStatistic(); // how many requests are being handled (full lifecycle)
     private final SampleStatistic _requestTimeStats = new SampleStatistic(); // latencies of requests (full lifecycle)
+    private final CounterStatistic _dispatchedStats = new CounterStatistic(); // how many requests are in handle
+    private final SampleStatistic _dispatchedTimeStats = new SampleStatistic(); // latencies of requests in handle
```

Comment:
Dispatched is too vague.  I think these should be called "Handling"

```suggestion
    private final CounterStatistic _handlingStats = new CounterStatistic(); // how many requests are in handle
    private final SampleStatistic _handlingTimeStats = new SampleStatistic(); // latencies of requests in handle
```

Reply:
Done.